### PR TITLE
Enhance typings for models and schema

### PR DESCRIPTION
**Pull request by hyochan**
- [x] Context typings
   - https://github.com/dotansimha/graphql-code-generator/issues/1090
- [x] Seqeulize class properties typings
   - https://sequelize.org/master/manual/typescript.html

### CODE REVIEW

File: src/models/Channel.ts [hunk 1]
```diff
@@ -36,4 +37,8 @@ Channel.belongsTo(User, {
   as: 'owner',
 });
 
-export default Channel;
+export type ChannelModelStatic = typeof Model & {
```
Comment (Sandwichj) [inline src/models/Channel.ts]: Looks good codes, just I didn't understand what does the `Static` mean? :?

Comment (hyochan) [reply src/models/Channel.ts]: You can read the above `sequelize` link I've mentioned.
image

Comment (hyochan) [reply src/models/Channel.ts]: Above also causes a problem when we are trying to pass `sequelize` definition through the graphql context.

Comment (Sandwichj) [reply src/models/Channel.ts]: Oh, sorry for that I missed the links. Thanks! :D

**Pull request merged**